### PR TITLE
Simplify fallback icon

### PR DESCRIPTION
**Pull request by osdnk**
Fixes RNBW-####

## Final checklist

- [ ] Assigned individual reviewers?
- [ ] Added labels?
- [ ] Added e2e tests? if not please specify why
- [ ] If you added new files, did you update the CODEOWNERS file?

### CODE REVIEW

File: src/components/coin-icon/CoinIconFallback.js [hunk 1]
```diff
@@ -22,65 +21,81 @@ const fallbackTextStyles = {
...
+    />
   );
-}
+});
+
+FallbackImage.displayName = 'FallbackImage';
+
+const WrappedFallbackImage = React.memo(
+  ({ color, elevation = 6, shadowOpacity, showImage, size, eth, ...props }) => {
+    const { colors } = useTheme();
```
Comment (terrysahaidak) [inline src/components/coin-icon/CoinIconFallback.js:62]: can we also pass "theme" down via props instead?

Comment (osdnk) [reply src/components/coin-icon/CoinIconFallback.js:62]: I think so? How to find a balance between what to pass thru props and what to evaluate in components?

File: src/utils/CoinIcons/CoinIcon.js [hunk 2]
```diff
@@ -26,31 +26,25 @@ const CoinIcon = ({
...
+    shadowOffset: {
+      height: isSmall ? 3 : 4,
+      width: 0,
+    },
+    shadowOpacity: isSmall ? 0.2 : 0.3,
+    shadowRadius: isSmall ? 4.5 : 6,
+  };
 
   const CoinIconElement = forceFallback
     ? fallbackRenderer
```
Comment (tchayen) [inline src/utils/CoinIcons/CoinIcon.js:50]: This file is now not used anymore and those changes don't really apply to fast icons.

File: src/utils/CoinIcons/FallbackIcon.js [hunk 3]
```diff
@@ -35,25 +36,24 @@ const FallbackIcon = ({
...
+          width,
+        },
+        style,
+      ]}
     >
-      <Text style={[sx.text, fontSize, textStyles]}>{formattedSymbol}</Text>
+      <Text style={[sx.text, { fontSize }, textStyles]}>{formattedSymbol}</Text>
     </View>
   );
 };
```
Comment (tchayen) [inline src/utils/CoinIcons/FallbackIcon.js:59]: This file also looks different post rewrite.

**Pull request closed without merging**